src/NServiceBus.Core/Serialization/SerializationSettingsExtensions.cs

```diff
@@ -2,13 +2,15 @@ namespace NServiceBus
 {
     using System;
     using System.Collections.Generic;
+    using NServiceBus.Logging;
     using Serialization;
     using Settings;
 
     static class SerializationSettingsExtensions
     {
         const string AdditionalSerializersSettingsKey = "AdditionalDeserializers";
         const string MainSerializerSettingsKey = "MainSerializer";
+        static readonly ILog log = LogManager.GetLogger(nameof(SerializationSettingsExtensions));
```

Comment:
Philosophical remark: We be nice to find a way in the future to get the logger in a non static way to make us more compatible with ILogger<T>